release_description.md

```diff
@@ -0,0 +1,31 @@
+The Amazon QLDB team is pleased to announce the release of v1.2.0 of the QLDB .NET driver.
+
+* Add the asynchronous version of the driver.
+* Improved retry logic to now handle more types of failure.
+
+The full list of changes are included in the [change log](https://github.com/awslabs/amazon-qldb-driver-dotnet/blob/master/CHANGELOG.md).
+
```

Comment:
Can we add a blurb saying the following snippets demonstrate how to use the async driver

Reply:
Done.